cql3/Cql.g

```diff
@@ -988,7 +988,7 @@ alterKeyspaceStatement returns [std::unique_ptr<cql3::statements::alter_keyspace
  * ALTER COLUMN FAMILY <CF> WITH <property> = <value>;
  * ALTER COLUMN FAMILY <CF> RENAME <column> TO <column>;
  */
-alterTableStatement returns [std::unique_ptr<alter_table_statement> expr]
+alterTableStatement returns [std::unique_ptr<alter_table_statement::raw_statement> expr]
```

Comment:
Please avoid having one-line commit messages (just a title), it's useful to explain not just what you did, but **why**. I am guessing that perhaps the reason why this "raw" thing was not needed earlier and needed now because now it's the first time that you allow prepared parameters (namely "USING TIMESTAMP ?") in an ALTER TABLE - which was already allowed in CREATE TABLE. If my guess is correct, please explain it. If it's incorrect, it's even more important to explain the correct explanation :-)